source/simulator/core.cc

```diff
@@ -1693,6 +1693,16 @@ namespace aspect
               solve_advection(AdvectionField::composition(c));
             }
 
+          if (
+            parameters.use_discontinuous_composition_discretization
+            &&
+            parameters.use_limiter_for_discontinuous_solution
+          )
+            {
+              pcout << " start to apply the limiter to the discontinuous solution "<< std::endl;
```

Comment:
I think we usually start with capital letters for screen output. I have not checked, but does this align nicely with the rest of our screen output? Minor detail: missing space before the last <<